spec/factories/notes.rb

```diff
@@ -53,7 +53,7 @@
     end
 
     trait :with_attachment do
-      attachment { fixture_file_upload(Rails.root + "spec/fixtures/dk.png", "`/png") }
```

Comment:
@Razer6 See my note about the test bug. This fix is recommended in https://github.com/activeadmin/activeadmin/issues/512